inductiva/admin/tasks.py

```diff
@@ -0,0 +1,43 @@
+"""Admin functions to manage tasks."""
+from typing import Dict, List, Optional
+
+import inductiva
+from inductiva import api
+from inductiva.client import ApiClient, ApiException
+from inductiva.client.apis.tags.admin_api import AdminApi
+from inductiva.client.model.task_status_code import TaskStatusCode
+
+
+def get_tasks(statuses: Optional[List[str]] = None,
+              usernames: Optional[List[str]] = None,
+              methods: Optional[List[str]] = None,
+              page=1,
+              per_page=10) -> List[Dict]:
+    api_config = api.validate_api_key(inductiva.api_key)
+
+    with ApiClient(api_config) as client:
+        api_instance = AdminApi(client)
+
+        query_params = {
```

Comment:
Potentially this could be a namedtuple, no? 
I would just like to avoid accessing fields using strings as dictionary keys, which is more error prone and the IDE can not provide validation for...

Reply:
Because that's the input that the generated client expects... this is how the docs generated by the client do it

Reply:
I am not sure how the namedtuple here would apply